scripts/startMongo.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+mongod --fork --logpath /var/log/mongod.log --replSet myapp --port 27017 --bind_ip 127.0.0.1,mongo
+mongo --eval 'rs.initiate({ _id: "myapp", members: [{ _id: 0, host: "mongo:27017"}] })'
+# we stared mongod in the background and don't want the container to exit
+tail -f /dev/null
```

Comment:
this is a new one to me!
I've seen idioms like `while : ; do done; `, maybe that is bad in terms of CPU cycles?

Reply:
I believe` tail -f /dev/null` in terms of CPU cycles over `while : ; do done;`, but never verified this. I've seen people just do `cat`. `tail -f /dev/null` was really confusing when i first saw it